_topic_map.yml

```diff
@@ -483,6 +483,8 @@ Topics:
     File: persistent-storage-cinder
   - Name: Persistent storage using Container Storage Interface (CSI)
     File: persistent-storage-csi
+  - Name: Persistent storage using CSI volume cloning
```

Comment:
> Since cloning (and snapshots) are available only in CSI, should it be a "child" of CSI chapter?

We aren't able to go another level in our TOC. However, we could create a separate h2 chapter under "Storage" called "Configuring persistent storage with CSI" and then include the chapter "[Persistent storage using the Container Storage Interface (CSI)](https://docs.openshift.com/container-platform/4.3/storage/persistent_storage/persistent-storage-csi.html)", along with CSI snapshots and CSI cloning snapshots.